extensions/security-jpa/deployment/pom.xml

```
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-security-jpa-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-security-jpa-deployment</artifactId>
+    <name>Quarkus - Security JPA - Deployment</name>
```

```suggestion
    <name>Quarkus - Elytron Security JPA - Deployment</name>
```
(btw, I agree it's ugly and nobody cares about Elytron but that's how all the others are)